src/constants/destinationCanonicalNames.js

```diff
@@ -1,5 +1,6 @@
 const DestHandlerMap = {
   ga360: 'ga',
+  salesforce_oauth: 'salesforce',
```

Comment:
Is it like a redirect to point to them OG salesforce destination even for the new Oauth destination?

Reply:
yes!!